src/frontend/Deprecation_analysis.ml

```diff
@@ -0,0 +1,168 @@
+open Core_kernel
+open Ast
+open Middle
+
+let deprecated_functions =
+  String.Map.of_alist_exn
+    [ ("multiply_log", "lmultiply")
+    ; ("binomial_coefficient_log", "lchoose")
+    ; ("integrate_ode", "integrate_ode_rk45")
```

Comment:
We need to expand/modify this:

integrate_ode -> ode_rk45
integrate_ode_rk45 -> ode_rk45
integrate_ode_bdf -> ode_bdf
integrate_ode_adams -> ode_adams

This was added recently (version 2.24).

Reply:
Thanks, updated.

Reply:
Thanks!